### PR TITLE
OCRAD-21 :- add concept class type document.

### PR DESCRIPTION
**Pull request by Akayeshmantha**

### CODE REVIEW

File: Concepts/concepts_class.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+# Concept Class Type
```
Comment (bmamlin) [inline Concepts/concepts_class.md]: `# Concept Class`

(remove "Type")

File: Concepts/concepts_class.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+In this way, the class is helpful for data extraction.  This classification details how a concept will be represented 
+(i.e. as a question or an answer) when the information is stored.  OpenMRS contains several default classes to use when 
+defining concepts, but implementation sites may also create additional custom concept classes for use. 
+
+## Available operations. 
+
+1. [List concept_class types](#list-concept-class-types)
+2. [Create a concept_class_type](#create-a-concept-class-type)
+3. [Update a concept_class type](#update-a-concept-class-type)
+4. [Delete a concept_class type](#delete-a-concept-class-type)
```
Comment (bmamlin) [inline Concepts/concepts_class.md]: Remove the term "type(s)"

* List concept classes
* Create a concept class
* Update a concept class
* Delete a concept class

File: Concepts/concepts_class.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    If user not logged in to perform this action,a `401 Unauthorized` status returned.
+
+    ##### Query Parameters
+
+    Parameter | Type | Description
+    --- | --- | ---
+    *q* | `Search Query` | Display Name of concept class Type.
+
+    ```console
+    GET /conceptclass?q="Search Query"
```
Comment (bmamlin) [inline Concepts/concepts_class.md]: Did this work for you? In my testing, using the `q` parameter doesn't work (I get an empty list back if `q` is set)

**Review by bmamlin — CHANGES_REQUESTED** [review]

**Comment by Akayeshmantha** [comment]
@bmamlin  done the changes here

**Pull request merged**